OpenEdXMobile/res/values/strings.xml

```diff
@@ -558,8 +558,8 @@
     <!-- User Profiles -->
     <!-- Window title for the user profile screen -->
     <string name="profile_title">Profile</string>
-    <!-- The accessibility description for the name section -->
-    <string name="profile_name_description">Name: {name}</string>
+    <!-- The accessibility description for the username section -->
+    <string name="profile_username_description">Username: {username}</string>
```

Comment:
@1zaman Also updated this. We have name and username. Name was being used here but the username was being displayed.